roles/pulp/README.md

```diff
@@ -109,6 +109,8 @@ This role is required by the `pulp_database` role and uses some variables from i
 
 * `pulp_settings_db_defaults`: See pulp_database README.
 
+* `pulp_ld_library_path`: When needed the LD_LIBRARY_PATH to be part of the environment for the pulpcore-api systemd process.
```

Comment:
What do you think about the following description instead?

An optional LD_LIBRARY_PATH environment variable for the pulpcore-api systemd process.